index.html

```diff
@@ -25,11 +25,188 @@ <h1>Kiprono Japhet</h1>
             </p>
         </div>
         
-        <button>Start collaboration</button>
+        <button type="submit">Start collaboration</button>
       </section>
         <div class="header-img-bottom">
             <img src="images/image_geometry_2.png" alt="image">
         </div>
     </header>
+  <section class="projects">
+    <div class="projects-wrapper">
+        <div class="project-image">
+            <img class="project-img" src="images/Rectangle 21 (1).png" alt="image">
+        </div>
+        <div class="project-lg">
+          <div class="project-lg-1">
+            <h3>Project name goes here</h3> 
+            
+            <button type="submit" class="project-btn-1"><span class="btn-p">HTML/CSS</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">Ruby on Rails</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">JavaScript</span></button>
+            
+            <button type="submit" id="project-view">
+                <span class="project-view-union">See this project</span>
+                
+                <span><img id="union" src="images/Union.png" alt="union"></span>
+            </button>
+          </div>
+        </div>
+    </div>
+    <div class="projects-wrapper">
+        <div class="project-image">
+            <img class="project-img" src="images/Rectangle 21.png" alt="image">
+        </div>
+        <div class="project-lg">
+          <div class="project-lg-1">
+            <h3>Project name goes here</h3> 
+            
+            <button type="submit" class="project-btn-1"><span class="btn-p">HTML/CSS</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">Ruby on Rails</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">JavaScript</span></button>
+            
+            <button type="submit" id="project-view">
+                <span class="project-view-union">See this project</span>
+                
+                <span><img id="union" src="images/Union.png" alt="union"></span>
+            </button>
+          </div>
+        </div>
+    </div>
+    <div class="projects-wrapper">
+        <div class="project-image">
+            <img class="project-img" src="images/Rectangle 21 (2).png" alt="image">
+        </div>
+        <div class="project-lg">
+          <div class="project-lg-1">
+            <h3>Project name goes here</h3> 
+            
+            <button type="submit" class="project-btn-1"><span class="btn-p">HTML/CSS</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">Ruby on Rails</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">JavaScript</span></button>
```

Comment:
- [ ] The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") should be list items `<li>` tags not `<button>` inside an unordered list `ul` tag as this is the right way ✔️  to markup a list of things.